docs/src/pages/playground.tsx

```diff
@@ -151,21 +102,19 @@ function Homepage() {
   }, []);
 
   return (
-    <div className={styles.page}>
-      <div className={styles.leftSide}>
-        <div className={styles.codeEditorBox}>
-          {showReactEditor ? (
-            <PlaygroundReactEditor code={code} onCodeChange={setCode} />
-          ) : (
-            <PlaygroundCodeEditor onChange={setScene} initialCodeEditorContent={INITIAL_SCENE} />
-          )}
-        </div>
+    <div className="flex flex-row flex-wrap p-8 lg:h-[calc(100vh-110px)]">
```

Comment:
unfortunately calculating size of a footer and nav bar is the easiest way, otherwise components deep inside the tree can easily force their size.